canonical/multipass.run
# Add RAD support for tabbed docs content

## QA

Check https://multipass-run-318.demos.haus/docs/set-up-the-driver has the tabbed content functionality.


![ezgif-5-d255494e97](https://github.com/canonical/multipass.run/assets/14939793/010071a5-e3bf-463c-9c50-76919c4c0aa2)
